test-five/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
```

Comment:
can you explain why you chose to introduce docker here?

Reply:
Docker references should not be present in this project because my solution does not utilise Docker. Initially, I intended to employ Docker for testing, deployment, and inclusion in the pipeline for deployment on Beanstalk or using Fargate. Consequently, I included Docker and Docker Compose files. However, considering the testing requirements, I decided to opt for an alternative architecture that does not rely on Docker. I forgot to delete those resources, even though it can still be used for local development.